crypto/cmp/cmp_msg.c

```diff
@@ -253,16 +253,16 @@ OSSL_CMP_MSG *ossl_cmp_msg_create(OSSL_CMP_CTX *ctx, int bodytype)
     (sk_GENERAL_NAME_num((ctx)->subjectAltNames) > 0 \
          || OSSL_CMP_CTX_reqExtensions_have_SAN(ctx) == 1)
 
-static const X509_NAME *determine_subj(OSSL_CMP_CTX *ctx,
-                                       const X509_NAME *ref_subj,
-                                       int for_KUR)
+static const X509_NAME *determine_subj(OSSL_CMP_CTX *ctx, int for_KUR,
+                                       const X509_NAME *ref_subj)
```

Comment:
Why change the order of the arguments?

Reply:
I changed the order because I realized that the new order is more logical:
The outcome of the local helper function `determine_subj()` 
* primarily depends on `ctx`, 
* then possibly on `int for_KUR`, 
* and then, in certain circumstances dependent on the other two params, it depends on `ref_subj`.